File: 7.Search-Algorithm/Example/Linear/Linear.go
package main

import "fmt"

func linearSearch(arr []int, num int) int {
	for i, v := range arr {
		if v == num {
			return i
		}
	}
	return -1
}
func main() {
	number := []int{10, 2, 4, 1, 5}
	fmt.Println(linearSearch(number, 4)) // 2
	fmt.Println(linearSearch(number, 6)) // -1
}

File: 1.Elementary level/7.Search-Algorithm/Example/Linear/Linear.py
number = [10,2,4,1,5]

def linearSearch(array,num):
    for i in range(len(array)):
        if array[i] == num:
            return i
    
    return -1

print(linearSearch(number,4)) # 2
print(linearSearch(number,6)) # -1